common/testData/datetime/README.md

```diff
@@ -11,4 +11,15 @@ and various input datetime values.
 The test generator constructs the expected value using the various pieces
 (date format pattern, time format pattern, datetime "glue" pattern)
 and `SimpleDateFormat`s to combine them together.
-Each test case reports the inputs for the test case and the expected value.
\ No newline at end of file
+Each test case reports the inputs for the test case and the expected value.
+
+The UTS 35 LDML spec for datetime formatting will be updated in CLDR v48 to specify
+that the default value for dateTimeFormatType will be "atTime".
+We use dateTimeFormatType to represent the value that indicates tthe type of
```

Comment:
Minor, but we tend to avoid the imperial "we". ;)

These can be simply restated as descriptions:

The dateTimeFormatType represents the value...

Reply:
Good point. I reworded accordingly.